src/translations.cpp

```diff
@@ -559,6 +633,20 @@ void translation::deserialize( JsonIn &jsin )
             // if plural form is enabled
             if( raw_pl ) {
```

Comment:
How will it have a raw_pl if "str_sp" is present? And "str_sp" indicates the automatic pluralization shouldn't happen, at all, since the strings are supposed to be the same?

Reply:
`str_sp` simply means the English singular and plural are the same, and it may still need different plurals in other languages, so it needs to have `raw_pl`.